### DISCUSSION TITLE
are our github account got deleted if our github education finished?

### DISCUSSION THREAD

**Discussion by ripaaf** [community/community#140156] [GitHub Education]
### Hi everyone,

as i wrote in the title is that true? i saw someone's post and he said it's a good idea to backup our repo because if it finished he said it will be gone..

**Comment by durugu-n** [comment]
Your account won't be deleted but your membership will be revoked to GitHub Free. Your repos will still be there but it's still a good idea to back them up whenever you can.

**Reply by ripaaf** [reply]
okay, thank you!
